src/App.js

```diff
@@ -1,42 +1,36 @@
+import {  Route, Routes } from "react-router-dom";
 import "./App.css";
-import logo from "./logo.png";
+import { Footer, Navbar } from "./components";
+import { Home,
+  Cart,
+  ProductListing,
+  SingleProduct,
+  Whishlist,
+  Login,
+  Signup,
+  Error
+} from "./pages";
+
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+    
+    <Navbar/>
+    <Routes>
+    <Route path='/' element={<Home/>}></Route>
+    <Route path='/products' element={<ProductListing/>}></Route>
+    <Route path="/cart" element={<Cart/>}></Route>
+    <Route path="/Wishlist" element={<Whishlist/>}></Route>
```

Comment:
keep the pathname consistent so it's easier to avoid mistakes, either all lowercase or the first letter capitalised for all paths.